tdrs-backend/plg/promtail/config.local.yml

```diff
@@ -24,6 +24,14 @@ scrape_configs:
         labels:
           job: django
           __path__: /logs/django.log
+    pipeline_stages:
```

Comment:
does this change only apply to local currently?

Reply:
We also need to extract these labels for every other job so that the new variables you added to the dashboard work for all log sources.

Reply:
Other jobs don't have these labels, for example the Nginx logs don't have level. One idea is to add level to Nginx logs, the easiest way would be to add level based on log type. For example, if it is Nginx access, then level can be info, if it is error_log, then it can be warning